app/i18n/locales/pt-BR.json

```diff
@@ -736,5 +736,7 @@
   "decline": "Recusar",
   "accept": "Aceitar",
   "Incoming_call_from": "Chamada recebida de",
-  "Call_started": "Chamada Iniciada"
+  "Call_started": "Chamada Iniciada",
```

Comment:
Not related to this PR, but this is new and doesn't follow the copy convention of sentence case.